examples/interpreter-in-browser/README.md

```diff
@@ -37,3 +41,50 @@ cd www
 npm run start
 open http://localhost:8080
 ```
+
+### Get Demo Running on OS X
+
+If you don't already have the Lumen dev environment and just want to play with the demo, do these steps, **_then_** continue to the [link package](#link-package) steps.
+
+Uninstall `rust` installed with Homebrew, so you can use `nightly`.
+
+```
+brew uninstall rust
+```
+
+Install `rustup` to manage your `rust` version.
+
+```
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+source ~/.cargo/env
+rustup target add wasm32-unknown-unknown --toolchain nightly
+cargo +nightly install wasm-bindgen-cli
+curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+wasm-pack build
+```
+
+## Compiling from Elixir to Erlang
+
+Install the mix decompile task:
+
+```
+mix archive.install github michalmuskala/decompile
+```
+
+Create a new mix project somewhere:
+
+```
+mix new ex_sample
+```
+
+You can use the `www/ex_sample.ex` file here for an example in your mix project lib folder. It shows some interaction with the DOM through `lumen_web`. Then run:
+
+```
+mix compile
+mix decompile --to-erl ExSample
```

Comment:
```suggestion
mix decompile ExSample --to erl
```

When I installed `decompile`, `--to-erl` was not a command line option and I need to do `--to<space>erl` instad